detect_alpine_image

```diff
@@ -6,7 +6,9 @@ img="unknown"
 if [ "$arch" = "x86_64" ]; then
     img="alpine"
 elif [ "$arch" = "armv7l" ]; then
-    img="arm32v6/alpine"
+    img="armhf/alpine"
```

Comment:
I think this PR shouldn't change the behaviour for other platforms. Probably an artifact of other parallel merges. Upstream already contains the changes for armv7l and aarch64.

Reply:
Indeed, that was the just solved merge conflict